donut/static/css/style.css

```diff
@@ -208,7 +207,7 @@ td {
   background-color: #FFF;
   color: #333;
   border-color: orange;
-  margin-bottom: 5%;
+  margin-bottom: 5px;
```

Comment:
Okay... but the other button classes and `:hover` variants still use `5%`, so when you hover over a button, it shrinks/expands. Also, can we please not apply these bottom margins inside `.input-group`s? They look pretty bad:
<img width="760" alt="Screen Shot 2020-02-23 at 13 02 17" src="https://user-images.githubusercontent.com/4256376/75120101-8bff9c80-563d-11ea-8e47-4d89625f5d90.png">
<img width="914" alt="Screen Shot 2020-02-23 at 13 16 02" src="https://user-images.githubusercontent.com/4256376/75120227-ca498b80-563e-11ea-8617-62ef5089059c.png">